code/blake2/Hacl.Impl.Blake2.Core.fst

```diff
@@ -31,6 +31,34 @@ let row_v #a #m h r =
 
 let row_v_lemma #a #m h0 h1 r1 r2 = ()
 
+let create_default_params a salt personal =
+  match a with
+  | Spec.Blake2S ->
+    Mkblake2s_params
```

Comment:
so I this done to avoid issues with field overloading, maybe there's a better issue... @mtzguido I thought you added something to help disambiguate field names when they're overloaded across two different types in scope?

Reply:
Indeed, if I add an extra type ascription, I can use the standard record syntax. I pushed an update.

Reply:
Sorry for the delay! I was away and just seeing this now. I didn't write that feature but yes, types can be used to disambiguate overloaded record fields.